rustler_bigint/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "rustler_bigint"
+version = "0.1.0"
```

Comment:
should this version be the same as `rustler` ? (or at least the minor version)

Reply:
Good question! I think we should use a different versioning, starting with `0.1.0`, as we probably want to indicate that this is new and not battle-tested yet. We can merge the versions when bumping rustler to `1.0`. Does that make sense?